frontend/src/old-pages/Configure/__tests__/useWizardNavigation.test.tsx

```diff
@@ -157,122 +157,6 @@ describe('Given a hook to navigate through the wizard', () => {
     })
   })
 
-  describe('when on cluster page', () => {
```

Comment:
question: the tests deleted from here, shouldn't they be still present somewhere else?

Reply:
It covered the `multiUser` case: if enabled, you navigate to multiUser page, headNode otherwise.

Moving multiUser to cluster page eliminates this edge case.
Standard flow has already been tested in other test cases.